mutt/date.h

```diff
@@ -26,6 +26,8 @@
 #include <stdbool.h>
 #include <time.h>
 
+#define MUTT_DATE_NOW 0
```

Comment:
Comment: `///< Constant representing the 'current time', see: mutt_date_gmtime(), mutt_date_localtime()`